code/game/world.dm

```diff
@@ -260,6 +264,11 @@ GLOBAL_VAR(restart_counter)
 	return
 	#endif
 
+	#ifdef AUTOWIKI
+	qdel(src)
```

Comment:
I don't think I ever actually call Reboot in AUTOWIKI, I qdel the world directly.